pkg/controllers/namespace/namespace_sync_controller.go

```diff
@@ -73,6 +73,13 @@ func (c *Controller) Reconcile(ctx context.Context, req controllerruntime.Reques
 		return controllerruntime.Result{Requeue: true}, err
 	}
 
+	if namespace.Labels != nil {
+		visibility, ok := namespace.Labels[policyv1alpha1.NamespaceAutoPropagationLabel]
+		if ok && visibility == "true" {
+			return controllerruntime.Result{}, nil
+		}
+	}
```

Comment:
```suggestion
	if strings.ToLower(util.GetLabelValue(namespace.Labels, policyv1alpha1.NamespaceAutoPropagationLabel)) == "true" {
		klog.Infof("Skip propagating namespace(%s) as per the constraint in label.", namespace.Name)
		return controllerruntime.Result{}, nil
	}
```
And move this before `List` clusters.

Reply:
Do you think the user must use the `true` in lowercase?